.github/workflows/kuksa_gps_feeder.yml

```diff
@@ -70,119 +72,30 @@ jobs:
         push: true
         tags: |
           ${{ steps.meta.outputs.tags }}
+          ${{ env.EPHEMERAL_IMAGE }}
```

Comment:
why do we add this?

Reply:
but there's no tag ```${{ env.EPHEMERAL_IMAGE }}``` in them or am I missing something?

Reply:
In [gps feeder](https://github.com/eclipse/kuksa.val.feeders/blob/main/.github/workflows/kuksa_gps_feeder.yml#L18) someone has by some reason defined an env variable:


```
env:
  EPHEMERAL_IMAGE: "ttl.sh/kuksa.val.feeders/gps-${{ github.sha }}"
```

which in this PR is used as

```
        tags: |
          ${{ steps.meta.outputs.tags }}
          ${{ env.EPHEMERAL_IMAGE }}
```

In the other repos no environment variable exists, instead the ttl.sh name is stated explicitly.



```
        tags: |
          ${{ steps.meta.outputs.tags }}
          ttl.sh/kuksa.val/kuksa-databroker-${{github.sha}}
```

```
        tags: |
          ${{ steps.meta.outputs.tags }}
          ttl.sh/kuksa.val/kuksa-dbcprovider-${{github.sha}}
```

So the end result shall be roughly the same, if you build on main/master/tag you will upload to both ttl.sh and ghcr.io. I however just noticed that the naming pattern differs for ttl.sh images, like if you compare `ttl.sh/kuksa.val/kuksa-dbcprovider-${{github.sha}}` with `ttl.sh/kuksa.val.feeders/gps-${{ github.sha }}`. But as we are about to move the feeders anyway that possibly does not make sense to fix.